db/data/20150311194246_associate_provers_with_ontology_versions.rb

```diff
@@ -9,4 +11,21 @@ def self.down
       ontology_version.save!
     end
   end
+
+  protected
+
+  def self.retry_provers_retrieval(ontology_version, max_attempts)
+    begin
```

Comment:
the begin is unnecessary. You can just issue the `rescue`s on the method's-body.